cli/crates/server/src/servers.rs

```diff
@@ -59,31 +58,35 @@ impl ProductionServer {
             bridge::build_router(message_sender.clone(), Arc::clone(&registry), tracing).await?;
         if !detected_udfs.is_empty() {
             validate_node().await?;
-            let project = Project::get();
-
-            let mut hasher = sha2::Sha256::new();
-
-            for entry in walkdir::WalkDir::new(&project.path)
-                .sort_by_file_name()
-                .follow_links(true)
-                .into_iter()
-                .filter_map(Result::ok)
-                // Only path we can somewhat safely ignore is the schema one
-                .filter(|entry| !entry.file_type().is_dir() && entry.path() != project.schema_path.path())
-            {
-                let content =
-                    std::fs::read(entry.path()).map_err(|err| ServerError::ReadFile(entry.path().into(), err))?;
-                hasher.update(entry.path().to_string_lossy().as_bytes());
-                hasher.update(content);
-            }
-            let hash = hasher.finalize().to_vec();
-            let hash_path = project.dot_grafbase_directory_path.join("grafbase_hash");
-            if hash != std::fs::read(&hash_path).unwrap_or_default() {
-                install_wrangler(Environment::get(), tracing).await?;
-                bridge_state.build_all_udfs(detected_udfs, parallelism).await?;
-            }
-            // If we fail to write the hash, we're just going to recompile the UDFs.
-            let _ = std::fs::write(hash_path, hash);
+            // TODO: the compile function also spawns Bun, we need to separate them if we want to do it conditionally
```

Comment:
Any reason to keep the commented code?

Reply:
We'll likely want to return it once we fix the structure